hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -278,7 +286,11 @@ void prepareComputation(final Bytes input, final UnaryOperator<byte[]> aliasReso
                             precompilePricingUtils,
                             functionId,
                             senderAddress,
-                            impliedTransfersMarshal);
+                            impliedTransfersMarshal,
+                            recordsHistorian,
+                            autoCreationLogic,
+                            dynamicProperties.isAutoCreationEnabled()
```

Comment:
_Nit_ - could add a `GlobalDynamicProperties.isImplicitCreationEnabled()` method to avoid repeating this conjunction in several places.